app/presenters/two_factor_options_presenter.rb

```diff
@@ -74,7 +75,14 @@ def piv_cac_option
 
   def backup_code_option
     policy = TwoFactorAuthentication::BackupCodePolicy.new(current_user)
-    return [TwoFactorAuthentication::BackupCodeSelectionPresenter.new] if policy.available?
-    []
+    if available_and_not_enabled?(policy)
```

Comment:
`availabe_and_not_enabled?` should probably be a method on the policy

Reply:
Bumping this comment. Whether a particular MFA option is allowed in a context is policy logic. The presenter should only be concerned about what to display here in response to a policy.

Reply:
Torn on this one.  An interface should generally be minimal and you are suggesting we provide a method which is a logical combination of two other methods.  Shall we provide all the variations?

Reply:
We should not provide everything, but we should with the ones we need.

In the policy, it may make sense to rename it to be something like `enrollabled?` instead of `available_and_not_enabled?` which is what the policy would be describing.

Reply:
Sounds good